conf/readinglist.ini

```diff
@@ -1,7 +1,7 @@
 [app:main]
 use = egg:readinglist
 pyramid.debug_notfound = true
-readinglist.backend = readinglist.backend.memory
+readinglist.backend = readinglist.backend.simpleredis
```

Comment:
This means that the README deserves a bit more of documentation :)

I suggest we leave memory by default but give details about changing conf when running Redis